src/main/ts/components/Editor.tsx

```diff
@@ -26,34 +26,114 @@ export type InitOptions = Omit<OmitStringIndexSignature<EditorOptions>, OmittedI
 export type Version = `${'4' | '5' | '6' | '7'}${'' | '-dev' | '-testing' | `.${number}` | `.${number}.${number}`}`;
 
 export interface IProps {
+  /**
+   * @see {@link https://www.tiny.cloud/docs/tinymce/latest/react-ref/#apikey React Tech Ref - apiKey}
```

Comment:
```suggestion
   * @see {@link https://www.tiny.cloud/docs/tinymce/7/react-ref/#apikey React Tech Ref - apiKey}
```
All links should point to v7 docs instead